openapi3/validation_options.go

```diff
@@ -92,6 +93,13 @@ func DisableExamplesValidation() ValidationOption {
 	}
 }
 
+// DisableVersionValidation disables openapi version validation.
+func DisableVersionValidation() ValidationOption {
```

Comment:
Please add the `Enable..` version as well.

Reply:
done